.github/workflows/spring-oracle-example.yml

```
@@ -0,0 +1,68 @@
+
+jobs:
+
+  integration-tests:
+    name: Run Unit & Integration Tests
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: spring-oracle-example
+    strategy:
+      matrix:
+        distribution: [ 'temurin' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)